server/src/main/java/io/druid/client/cache/CacheConfig.java

```diff
@@ -52,10 +52,13 @@
   private int cacheBulkMergeLimit = Integer.MAX_VALUE;
 
   @JsonProperty
-  private int resultLevelCacheLimit = Integer.MAX_VALUE;
+  private int maxEntrySize = 1_000_000;
```

Comment:
What is a _typical_ cache entry size? (how did you pick this number)

Reply:
In our cluster the average size is about 1KB. But I am not sure how representative this is. I chose 1MB because I figured it was a large number that will block egregiously large cache entries.